grid2op/gym_compat/gym_act_space.py

```diff
@@ -115,11 +116,27 @@ class __AuxGymActionSpace:
     }
     keys_human_2_grid2op = {v: k for k, v in keys_grid2op_2_human.items()}
 
-    def __init__(self, env, converter=None, dict_variables=None):
+
+    def __init__(
+        self,
+        env,
+        converter=None,
+        dict_variables=None,
```

Comment:
Isn't this variable, named `dict_variables` doing what you want to do ? (if so it would be clearer to just rename it, no ?)

Reply:
same for observation by the way :-)